config/dr_cluster/manifests/ramen/kustomization.yaml

```diff
@@ -1,10 +1,10 @@
 # These resources constitute the fully configured set of manifests
 # used to generate the 'manifests/' directory in a bundle.
 resources:
-- bases/ramen_hub.clusterserviceversion.yaml
-- ../default
-- ../samples
-- ../../scorecard
+- ../bases/ramen_dr_cluster.clusterserviceversion.yaml
```

Comment:
files got flipped, should be `ramen_hub` here instead of `ramen_dr_cluster`? (vice versa in the other kustomization file?

Reply:
ugh! git diff renamed the files differently ` config/hub/manifests/kustomization.yaml → config/dr_cluster/manifests/ramen/kustomization.yaml` So this should be dr-cluster as in here :)

Reply:
The git matching algorithm decided that the match percentage is more which looking at the other file. IOW, the diff is showing *both* the path and the content changed.